integration/nwo/chaincode_client_template.go

```diff
@@ -0,0 +1,18 @@
+/*
```

Comment:
This really shouldn't be a template as it's not part of the configuration, it's part of the requirement for a single chaincode implementation.

Also, once `ChaincodeServerAddress` is removed from the config (since it shouldn't be there), there will be nothing to template against.